template/cordova/lib/MSBuildTools.js

```diff
@@ -46,11 +46,8 @@ MSBuildTools.prototype.buildProject = function(projFile, buildType, buildarch, o
     '/p:Configuration=' + buildType,
     '/p:Platform=' + buildarch];
 
-    if (otherConfigProperties) {
-        var keys = Object.keys(otherConfigProperties);
-        keys.forEach(function(key) {
-            args.push('/p:' + key + '=' + otherConfigProperties[key]);
```

Comment:
Also it's expected that user should provide already prefixed flags (since `/p:` is not the only prefix available)